src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetError/WidgetError.types.ts

```diff
@@ -0,0 +1,10 @@
+import type { Message } from '../content/WidgetErrorAndWarningContent.types';
+
+export type Props = {
+    error?: Array<Message>;
+};
+
+export type PlainProps = {
+    error?: Array<Message>;
+    classes: string;
+};
```

Comment:
Delete `PlainProps` and import Props in src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetWarning/WidgetWarning.component.tsx